src/content/config.ts

```diff
@@ -91,8 +91,9 @@ export const collections = {
 			description: z.string(),
 			publishDate: z
 				.string()
```

Comment:
This must have either been before `z.date({ coerce: true })` was a thing *or* before I realized the coerce option existed 😅 

We could probably simplify this to avoid the `string || date` + `transform` steps